### PR TITLE
add allahcalculator (lite version)

### PR DESCRIPTION
**Pull request by clyde37**
This is the lite version that does not require tons of space and processing power

### CODE REVIEW

File: allahcalc.mk [hunk 1]
```diff
@@ -0,0 +1,26 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS += allahcalc
+ALLAHCALC_VERSION := 1.1
+DEB_ALLAHCALC_V ?= $(ALLAHCALC_VERSION)
+
+allahcalc-setup: setup
+	wget -q -nc -P (BUILD_SOURCE) https://github.com/clyde37/allah-calculator/releases/download/$(ALLAHCALC_VERSION)/allah_calculator_generator.cpp
```
Comment (TheRealKeto) [inline allahcalc.mk:10]: Use the ``GITHUB_ARCHIVE`` and ``EXTRACT_TAR`` functions, instead.

File: allahcalc.mk [hunk 2]
```diff
@@ -0,0 +1,26 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS += allahcalc
+ALLAHCALC_VERSION := 1.1
+DEB_ALLAHCALC_V ?= $(ALLAHCALC_VERSION)
```
Comment (TheRealKeto) [inline allahcalc.mk:7]: Fix your var spacing.

File: allahcalc.mk [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+	wget -q -nc -P (BUILD_SOURCE) https://github.com/clyde37/allah-calculator/releases/download/$(ALLAHCALC_VERSION)/allah_calculator_generator.cpp
+
+ifneq ($(wildcard $(BUILD_WORK)/allahcalc/.build_complete),)
+allahcalc:
+	@echo "Using previously built allahcalc."
+else
+allahcalc: allahcalc-setup
+    gcc++ allah_calculator_generator.cpp -o allahcalc/allah_calculator_generator
+    ./allah_calculator_generator
+    gcc allah_calculator_final.mp4_custom.c -o allahcalc/allahcalc
```
Comment (TheRealKeto) [inline allahcalc.mk:19]: Compile this with clang. Figure it out yourself.

File: allahcalc.mk [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+endif
+
+allahcalc-package: allahcalc-stage
+    $(call PACK,allahcalc,DEB_ALLAHCALC_V)
+    rm -rf allahcalc
```
Comment (TheRealKeto) [inline allahcalc.mk:24]: You should know where files get put, so fix this to use the proper directories.

**Review by TheRealKeto — CHANGES_REQUESTED** [review]
This PR seems rushed. You need a control file, and rebase to ``as2``, instead of ``master``.

File: allahcalc.mk (hunk 3 above)
Comment (clyde37) [reply allahcalc.mk:19]: allah calc doesnt work with clang, at least the second part doesnt.

Comment (TheRealKeto) [reply allahcalc.mk:19]: Then, you can't even add this to Procursus. We don't have gcc.

File: allahcalc.mk (hunk 4 above)
Comment (clyde37) [reply allahcalc.mk:24]: not sure what that means

File: allahcalc.mk (hunk 1 above)
Comment (clyde37) [reply allahcalc.mk:10]: I'm not sure why you would do that, but sure I can change it to work that way

File: allahcalc.mk (hunk 4 above)
Comment (TheRealKeto) [reply allahcalc.mk:24]: If you know what ``$(BUILD_SOURCE)`` is for, you should know that, as of right now, this entire thing won't compile at all. You need to move your files to specific places before compiling them.

File: allahcalc.mk (hunk 3 above)
Comment (clyde37) [reply allahcalc.mk]: it works in lite mode

File: allahcalc.mk [hunk 5]
```diff
@@ -14,7 +14,7 @@ allahcalc:
 	@echo "Using previously built allahcalc."
 else
 allahcalc: allahcalc-setup
-    gcc++ allah_calculator_generator.cpp -o allahcalc/allah_calculator_generator
+    clang++ allah_calculator_generator.cpp -o allahcalc/allah_calculator_generator
```
[the remark this answers is not in the document]
Comment (clyde37) [reply allahcalc.mk:17]: sure

**Comment by CRKatri** [comment]
Closing this cause it’s a troll PR.

**Comment by clyde37** [comment]
how is it a troll pr? I spent over 30 minutes trying to figure out how to understand these makefiles

**Comment by Gymxo** [comment]
> how is it a troll pr? I spent over 30 minutes trying to figure out how to understand these makefiles

Troll

**Comment by demhademha** [comment]
To be honest, I don't think the nature of this package (particularly its name) is appropriate.

**Comment by demhademha** [comment]
I have also just checked the source code, 
https://github.com/clyde37/allah-calculator/blob/main/allah_calculator_generator.cpp
Not only is it full of **religious slurs**, but the   license prevents the package from being distributed.

**Comment by Gymxo** [comment]
Definitely a troll

**Comment by proprdev** [comment]
We do a minuscule amount of trolling

**Pull request closed without merging**